acme/acme.go

```diff
@@ -549,16 +593,36 @@ func (a *ACME) LoadCertificateForDomains(domains []string) {
 		} else {
 			domain = Domain{Main: domains[0]}
 		}
+
 		if _, exists := account.DomainsCertificate.exists(domain); exists {
-			// domain already exists
+			log.Debugf("ACME Certificate for domain %v exists.", domain)
 			return
 		}
+
+		// Check if our domains are matching our ingoreFilters
+		if len(a.IgnoreFilters) > 0 {
```

Comment:
is `loadCertificateOnDemand` the correct location for this code?

This means that the certificate will be requested from acme, but not returned correct?

Wouldn't this be better served in `getCertificate` before a potential cert request?